services/bit/bit-components.tester.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const Joi = require('joi')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('collection (valid)')
+  .get('/ramda/ramda.json')
+  .expectBadge({
+    label: 'components',
+    message: Joi.string().regex(/^[0-9]+$/),
+  })
+
+t.create('collection (valid)')
+  .get('/bit/no-collection-test.json')
+  .expectBadge({ label: 'components', message: 'not found' })
```

Comment:
The expected message here will also need to be updated to reflect the change made to the 404 error 👍